### PR TITLE
county level scraper for MO

### PR DESCRIPTION
**Pull request by paulboal**
This is a county-level scraper for MO from the public health website that was already documented: https://health.mo.gov/living/healthcondiseases/communicable/novel-coronavirus/

### CODE REVIEW

File: scrapers/USA/MO/index.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+    const $trs = $table.find('tr:not(:first-child)');
+    $trs.each((index, tr) => {
+      const $tr = $(tr);
+      let countyName = parse.string($tr.find('td:nth-child(1)').text());
+      countyName = this._countyMap[countyName] || countyName;
+
+      const casesState = parse.number($tr.find('td:nth-child(2)').text()) || 0;
+      const casesOther = parse.number($tr.find('td:nth-child(3)').text()) || 0;
+      if (countyName !== 'TBD') {
```
Comment (lazd) [inline scrapers/USA/MO/index.js]: For `TBD`, can you add it to another object `{ county: UNASSIGNED, cases: cases)`?

File: scrapers/USA/MO/index.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+          counties[countyName] += casesState + casesOther;
+        } else {
+          counties[countyName] = casesState + casesOther;
+        }
+      }
+    });
+
+    const countiesList = [];
+
+    for (const c in counties) {
```
Comment (lazd) [inline scrapers/USA/MO/index.js]: `transform.objectToArray()`

File: scrapers/USA/MO/index.js (hunk 1 above)
Comment (paulboal) [reply scrapers/USA/MO/index.js]: Got it.  Done.

File: scrapers/USA/MO/index.js (hunk 2 above)
Comment (paulboal) [reply scrapers/USA/MO/index.js]: Didn't see that.  Great! and Done.

File: scrapers/USA/MO/index.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+    $trs.each((index, tr) => {
+      const $tr = $(tr);
+      let countyName = parse.string($tr.find('td:nth-child(1)').text());
+      countyName = this._countyMap[countyName] || countyName;
+
+      const casesState = parse.number($tr.find('td:nth-child(2)').text()) || 0;
+      const casesOther = parse.number($tr.find('td:nth-child(3)').text()) || 0;
+      if (countyName !== 'UNASSIGNED') {
+        countyName = transform.addCounty(countyName);
+      }
```
Comment (lazd) [inline scrapers/USA/MO/index.js]: So for this, you previously had stuff matching on `TBD` -- we don't want to ignore that, it needs to be added into a separate object with `{ county: UNASSIGNED }` -- the constant from above (see the code comment at the top of the file).

Comment (paulboal) [reply scrapers/USA/MO/index.js]: Thanks for the clarification.  Now I get it.

**Comment by lazd** [comment]
In my haste of merging your excellent contribution, I forgot to ask that you add yourself as the maintainer if you'd like to be credited on the site and are willing to maintain it (like done here): https://github.com/lazd/coronadatascraper/blob/master/scrapers/BRA/index.js#L15-L24

Additionally, we've also added a `source` field where we put the `name`, `url` or anything else that's relevant so we can cite the source properly. I see that, for this PR, your URL is the same as your source (for others, the source may be, say, the Department of Health, but the URL is ArcGIS's server). In this case, I would add:

```
source:  {
  name: 'Missouri Department of Health and Senior Services'
}
```

**Pull request merged**